crates/pop-wallet-integration/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "pop-wallet-integration"
```

Comment:
To be honest, not much additional thought was given besides following existing architecture. I am sure there are things that this could be extended into in terms of (potentially) hosting the backend as an API, extending to other types of signing, etc.

But, the maintenance burden is valid. I will move to a sub-module within the pop-cli crate. 

Feel free to mark as resolved if satisfied. cc @AlexD10S

Reply:
Please see https://github.com/r0gue-io/pop-cli/pull/362/commits/8cffac1ebd896e63db3a5eb08d399505001d5939. Resolve if happy.

Reply:
Joining the discussion a bit late here, but I like the last change you made. 
If we believe other projects can use this functionality, we can place it in a separate crate, but for now is fine in pop-cli crate.